lerna.json

```diff
@@ -1,7 +1,8 @@
 {
   "packages": [
     "packages/cactus-*",
-    "examples/cactus-*"
+    "examples/cactus-*",
+    "extensions/htlc/*"
```

Comment:
@jordigiam Sorry I think I forgot to update this yesterday. If you change it to `extensions/*` then lerna should start picking up the htlc packages as well (I'm guessing right now they just get ignored by the build because after me having flattened the directory structure this pattern no longer matches them)

Reply:
Done